authors/apps/articles/serializers.py

```diff
@@ -210,3 +210,12 @@ class ReportArticlesSerializer(serializers.ModelSerializer):
     class Meta:
         model = ReportArticle
         fields = ('article', 'user', 'report_msg')
+
+        
+class EmailCheckSerializer(serializers.Serializer):
+    email = serializers.EmailField(
+        required=True,
+        error_messages={
+            'invalid': 'Must be of email format'
```

Comment:
What's the email format? Make this more descriptive

Reply:
on it